src/views/Compose.vue

```diff
@@ -680,6 +694,26 @@ export default {
       // TODO: To release dithering, update the line below to remove the is_staff check;`
       return this.$store.state.profile.is_staff && !this.simpleInterface && instrumentCategory === 'IMAGE';
     },
+    mosaicAllowed: function(request) {
+      if (request.configurations.length !== 1) {
+        return false;
+      } else if (_.get(this.instruments, [request.configurations[0].instrument_type, 'type']) !== 'IMAGE') {
+        return false;
+      } else if (request.configurations[0].target.type !== 'ICRS') {
+        return false;
+      }
+      // TODO: To release mosaicing, update the line below to remove the is_staff check`
+      return this.$store.state.profile.is_staff && !this.simpleInterface;
+    },
+    extraMosaicRotation: function(configuration) {
+      let rotatorAngle = 0;
+      for (let instConfigIndex in configuration.instrument_configs) {
+        if ('extra_params' in configuration.instrument_configs[instConfigIndex]) {
```

Comment:
I wonder if we should also check that it has a rotator mode set... if not we should probably ignore a rotator_angle in the extra params

Reply:
Good idea. I added a check in to make sure the SKY rotator mode is set before applying rotation.